packages/react/ds/src/combo-box/dropdown-item.tsx

```diff
@@ -82,15 +84,13 @@ export const DropdownItem = ({
         </div>
 
         <Icon
-          className={`${isOpen && 'gi-rotate-180'}`}
+          className={isOpen ? 'gi-rotate-180' : ''}
```

Comment:
to keep consistency you can have something like: 

`cn({` 'gi-rotate-180': isOpen })